cc_library/py-scc/src/sciler/scclib/app.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, config, device):
             os.mkdir("logs")
         filename = "logs/log-" + datetime.now().strftime("%d-%m-%YT--%H-%M-%S") + ".txt"
         logging.basicConfig(
-            level=logging.INFO,
+            level=logging.FATAL,
```

Comment:
Maybe change this back since the written logs will also not record everything anymore.